.proxyrc.json

```diff
@@ -4,6 +4,13 @@
     "target": "http://127.0.0.1:8002/",
     "timeout": 30000
   },
+  "/versions/v6.0.0/backend": {
+    "target": "http://127.0.0.1:8002/",
+    "pathRewrite": {
+      "^/versions/v6.0.0/backend": ""
+    },
+    "timeout": 30000
+  },
```

Comment:
wow :( I suppose we don't have many more options for a quick new release

Reply:
Yeah but that means docker-compose, and… well, it does mean docker-compose 🫠

Reply:
(we should investigate further but let's land this patch and release a new version first)

Reply:
Sure, but we already use docker compose for PG and it works pretty well for now 😊 But yes, let’s land it as it is for now!